manifests/host.pp

```diff
@@ -1,17 +1,13 @@
 # == Define: dhcp::host
 #
 define dhcp::host (
-  $ip,
-  $mac,
-  $options = {},
-  $comment='',
-  $ignored = false,
+  Stdlib::Compat::Ipv4 $ip,
+  String $mac,
```

Comment:
Maybe, but there are multiple ways of noting it down. Some prefer dashes over colons and I couldn't find if dhcpd accepts those as well. The stackoverflow ones also requires 2 chars (so 00 over 0) where dhcpd accepts one as well. Then again, I think a common validation that accepts a bit more than dhcpd accepts might not be the worst thing.